### PR TITLE
Fixed JSDoc and updated syntax to reflect es2015

### PR DESCRIPTION
**Pull request by EricRabil**

### CODE REVIEW

File: src/Node.js [hunk 1]
```diff
@@ -71,18 +97,18 @@ module.exports = class Node {
...
       switch(message.op) {
-      case "sendWS": {
-        this.master.client.ws.send(message.d)
-        console.log("We need to send a WS!!!", message)
-        break
-      }
-      case "stats": {
-        this.stats = message.d
-      }
+        case "sendWS": {
```
Comment (Mstrodl) [inline src/Node.js]: Please don't indent case blocks

File: src/Player.js [hunk 2]
```diff
@@ -1,21 +1,64 @@
...
- * @prop {Boolean} ready Whether or not the node is connected to the voice channel
- * @prop {Object} nowPlaying The currently playing track's info
- */
+const EventEmitter = require("events").EventEmitter
+
 module.exports = class Player extends EventEmitter {
+  /**
+   * Represents a Player system
+   * @extends EventEmitter
+   * @prop {VoiceChannel} channel The voice channel the player is playing to
```
Comment (Mstrodl) [inline src/Player.js]: You forgot to remove this from the top after you moved it into the constructor body

File: src/Player.js [hunk 3]
```diff
@@ -176,42 +192,42 @@ module.exports = class Player extends EventEmitter {
...
-    }
-    case "trackEnd": {
-      this.nowPlaying = null
-      /**
-       * Fires when a track ends
-       * @event Player#end
-       */
-      this.emit("end")
-    }
+      case "connected": {
```
Comment (Mstrodl) [inline src/Player.js]: Same here, don't indent case blocks

**Review by Mstrodl — CHANGES_REQUESTED** [review]
Looks good, just a few things to fix before I merge

File: src/Player.js [hunk 4]
```diff
@@ -1,9 +1,13 @@
 const EventEmitter = require("events").EventEmitter
 
-module.exports = class Player extends EventEmitter {
+/**
```
Comment (Mstrodl) [inline src/Player.js]: Not what I meant. You forgot to remove the `@prop`  part

**Review by Mstrodl — CHANGES_REQUESTED** [review]

File: src/OofClient.js [hunk 5]
```diff
@@ -7,16 +7,35 @@ const REGION_MAP = {
...
+     * The Discord Client this OofClient is for
+     * 
+     * @type {Client}
+     */
     this.client = client
+
+    /**
+     * The players for each guild
+     * 
+     * @type {Guild}
```
Comment (Mstrodl) [inline src/OofClient.js:37]: Shouldn't this be `{Object[Guild]}`?

**Review by Mstrodl — CHANGES_REQUESTED** [review]

**Review by Mstrodl — APPROVED** [review]
Thanks! :)

**Comment by EricRabil** [comment]
merg

**Pull request merged**